test/node-js-subsystem-labels.test.js

```diff
@@ -130,3 +130,13 @@ tap.test('label: lib internals without "lib / src" limiting', (t) => {
 
   t.end()
 })
+
+tap.test('label: add subsystem when ./doc/api/<subsystem>.md has been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'doc/api/fs.md'
+  ], false)
+
+  t.same(labels, ['doc', 'fs'])
```

Comment:
What labels should be added if `./doc/api/fs.md` *and* `./doc/api/http.md` had been changed in the same PR?